connector/checkpoint/checkpoint.py

```diff
@@ -184,6 +184,8 @@ def _needaction_domain_get(self):
         return [('state', '=', 'need_review')]
 
 
+# TODO: deprecate these since we can now add checkpoints
```

Comment:
It would be nice if there were a `_logger.warn(` to indicate the deprecation log-side